_includes/specials/taproot/en/01-single-sig.md

```diff
@@ -0,0 +1,75 @@
+Using Optech's [transaction size calculator][], we can compare the sizes
+of different types of single-sig transactions.  As expected,
+transactions using P2WPKH inputs and outputs are much smaller than those
+using P2PKH inputs and outputs---but, perhaps surprisingly, P2TR
+transactions are slightly larger than equivalent P2WPKH transactions.
+
+|                    | P2PKH (legacy) | P2WPKH (segwit v0) | P2TR (taproot/segwit v1) |
+|--------------------|----------------|--------------------|--------------------------|
+| **Output**         |  34            | 31                 | 43                       |
+| **Input**          |  148           | 68                 | 57.5                     |
+| **2-in, 2-out tx** |  374           | 208.5              | 211.5                    |
```

Comment:
Maybe worth to also list the sizes for a 2-in, 3-out transaction, where Taproot does become favorable?

Reply:
@shesek do you mean 3-in, 2-out?

I think I prefer to stick with the apples-to-apples comparison and link to the calculator so that anyone who wants to can play around with the numbers however they'd like.  Including more examples in the table might confuse readers and we directly address the issue of taproot inputs being cheaper in the first bullet point, so hopefully nobody will be missing the overall point.

Reply:
"2-in, 2-out" is a good example case, no need to sugarcoat. The important parts regarding Taproot's weight increase are that the overall cost increase is minuscule, the input to output costs have better incentive alignment (increasing the UTXO set size is more expensive than for other single-sig types, but reducing it is cheaper), and, as @harding writes in this section, _the receiver picks the destination address_, but the sender pays for the output that pays to it and the receiver pays for the subsequent input. Hence, the receiver is incentivized to pick P2TR as their address to get paid to.